README.md

```diff
@@ -57,8 +57,11 @@ You can run the software directly using Docker.
 
 1. [Install Docker](https://docs.docker.com/engine/install/), then start Docker locally.
 2. Run `docker pull sweagent/swe-agent:latest`
-3. Add your API tokens to a file `keys.cfg` as explained [below](#-add-your-api-keystokens)
+3. Copy the `example_cfg.key` to `keys.cfg` and add your API tokens as explained [below](#-add-your-api-keystokens). You can do this with the following command:
```

Comment:
In the the express setup, users don't have a local copy of the repository, though, so that doesn't work.